### ISSUE TITLE
Docs: state.State.includes description wrong

### ISSUE THREAD

**Issue by RobYed** [angular-ui/ui-router#3356]
This is a:

- [x] Bug Report
- [ ] Feature Request
- [ ] General Query 

My version of UI-Router is: 1.0.0-rc.1

# Bug Report

#### Current Behavior:

I was wondering what the docs say about the shape and behaviour of [`state.State.includes`](https://ui-router.github.io/ng1/docs/1.0.0-rc.1/classes/state.state-1.html#includes). I found this:

> An array of strings of the parent States' names

Actually it is not an array, but an object, containing the parent States' names. The TypeScript definition is `{ [name: string] : boolean }`.

#### Expected Behavior:

The docs should fit to the actual implementation (of course). Furthermore I expected the docs to tell me that the boolean value is always `true` for any key in the object. I did not go through the code to verify that, but I could not imagine a case where it should not be `true`.

To sum up, I'd expect to read something like this in the docs:

> An object, containing the parent States' names as keys and `true` as their values

If you are okay with this, I could open up a PR. Just let me know.

I also don't want to miss the opportunity to thank you for this amazing lib! My employer is using UI Router 1.0.0-rc.1 for AngularJS in several production applications. We did not have any problems with it. The new version is a great improvement and helps a lot in building component based apps!

**Comment by christopherthielen** [comment]
I would appreciate that PR!

You could also add an example, such as:

```js
(to: StateDeclaration) => to.$$state().includes['parent']
```

**Comment by christopherthielen** [comment]
Closed by https://github.com/ui-router/core/pull/48
